browser/android/preferences/brave_pref_service_bridge.cc

```diff
@@ -175,5 +210,19 @@ jboolean JNI_BravePrefServiceBridge_GetUseRewardsStagingServer(JNIEnv* env) {
       brave_rewards::prefs::kUseRewardsStagingServer);
 }
 
+jboolean JNI_BravePrefServiceBridge_GetBooleanForContentSetting(JNIEnv* env, jint type) {
```

Comment:
this should return the java ContentSetting class. It's less confusing than changing types and there are other options for the default like CONTENT_SETTING_ASK

Reply:
we are setting the default values with a new methods, we just need to check bools here. Marking as resolving as discussed that via DM